tests/resources/apiritif/test_codegen_requests.py

```diff
@@ -8,15 +8,14 @@
 
 import apiritif
 
-vars = {
-    
-}
-
 
 class TestAPI(unittest.TestCase):
-    
+
+    def setUp(self):
+        self.vars = {
```

Comment:
Wait a second. If we define `vars` as a property of TestCase and clear them each time we do `setUp()` — they will be cleaned after each test method is called. This will probably break extractors. Did you check for that?

Reply:
Currently, we rely on fact of generating single test method. Otherwise there is conflict of global vars being shared between threads.

Reply:
setUpClass will be better place..

Reply:
`setUpClass` has no object instance, we already bumped into this